docs/install/ubuntu_setup.md

```diff
@@ -170,44 +191,72 @@ If building on CPU and using OpenBLAS:
 ```bash
     git clone --recursive https://github.com/apache/incubator-mxnet.git
     cd incubator-mxnet
-    echo "USE_OPENCV = 1" >> ./config.mk
-    echo "USE_BLAS = openblas" >> ./config.mk
-    make -j $(nproc)
+```
+
+
+```bash
+    rm -rf build
+    mkdir -p build && cd build
+    cmake -GNinja \
+        -DUSE_CUDA=OFF \
+        -DUSE_MKL_IF_AVAILABLE=OFF \
+        -DCMAKE_CUDA_COMPILER_LAUNCHER=ccache \
+        -DCMAKE_C_COMPILER_LAUNCHER=ccache \
+        -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
+    ..
+    ninja
```

Comment:
`-DUSE_MKLDNN` will be turned on by default in this cmake line? Seems we need change the description in the next section and document the default behavior somewhere.

Reply:
MKL install requires additional steps in Ubuntu that are not part of the installation, so it's disabled. by -DUSE_MKL_IF_AVAILABLE=OFF even if any other MKL variables are on.

Reply:
@TaoLv happy if you can help with MKL and CMake: https://github.com/apache/incubator-mxnet/issues/14670

Reply:
All MKL is disabled, check the end of the line:  https://github.com/apache/incubator-mxnet/blob/master/CMakeLists.txt#L31